### PR TITLE
Update docs for cloud services and SSO

### PR DESCRIPTION
**Pull request by tlwr**
What
----

I have refactored the "Manage access to AWS" to be "Manage access to Cloud services" and added non-exhaustive info about

- Google Cloud
- Aiven
- Microsoft Azure

I have also added some very light documentation about rotating UAA SSO OAuth credentials.

How to review
-------------

`bundle install && rake build && (cd build && python -m http.server)`
`open http://localhost:8000`

Who can review
--------------

Not @tlwr

### CODE REVIEW

File: source/team/rotating_credentials.html.md [hunk 1]
```diff
@@ -55,6 +55,24 @@ The following is a diagram of the steps of our CA rotation process. The thick ar
...
+
+We use single sign-on using Google and Microsoft to allow our tenants to sign
+in, without using a username and password stored in UAA.
+
+Google has two credentials: `client_id` and `client_secret`
+
+Microsoft functionally has three credentials `client_id`, `client_secret`, and
+`tenant_id`, however `tenant_id` is not secret, we just store it as such.
+
+UAA is not able of consuming multiple pairs of `client_id` and `client_secret`.
```
Comment (AP-Hunt) [inline source/team/rotating_credentials.html.md]: *capable?

**Comment by AP-Hunt** [comment]
As a side note, `bundle exec middleman` is the easier way of building and running the generated site

File: source/team/managing_access_to_cloud_services.html.md [hunk 2]
```diff
@@ -1,22 +1,61 @@
...
+We have 4 separate projects:
+
+- Dev
+- CI
+- Staging
+- Prod
+
+# Microsoft Azure
+
+We use Microsoft Azure for single sign-on. We log into Azure using the [Azure
```
Comment (46bit) [inline source/team/managing_access_to_cloud_services.html.md]: To clarify who uses it:
```suggestion
We use Microsoft Azure to provide single sign-on for some tenants. We log into Azure using the [Azure
```

File: source/team/rotating_credentials.html.md (hunk 1 above)
Comment (tlwr) [inline source/team/rotating_credentials.html.md]:
```suggestion
UAA is not capable of consuming multiple pairs of `client_id` and `client_secret`.
```

**Pull request merged**